### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ogchy**
- [DEMO LINK](https://ogchy.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ogchy.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,50 @@
...
+        >
+      </div>
+
+      <div class="card__description">
+        <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      </div>
+
+      <div class="card__product">
+        <h3>Product code: 195434</h3>
+      </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:26]:
```suggestion
      
        <h3 "card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
      
        <p class="card__product">Product code: 195434</p>
```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,50 @@
...
+
+      <div class="stars">
+        <div class="stars__raiting">
+          <img src=".//images/stars.png" alt="stars">
```
Comment (SanyaBratashchuk) [inline src/index.html:30]: image

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,50 @@
...
+        </div>
+
+        <div class="stars__reviews">
+          <h4>Reviews: 5</h4>
+        </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:35]:
```suggestion
          <p class="card__reviews">Reviews: 5</p>
```

File: src/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/style.css:10]: add paddings, border for card

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
check font-sizes, colors, intends through the project

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
1) Use nesting in the SCSS files
2) As was told before, check all the fonts, font-sizes and coloures of the borders. Everything should be as in the Figma.

**Review by l4st1m0za — CHANGES_REQUESTED** [review]
These are differences from the first test
image

And these are from the second test
image

You need to pay attention to the: font, font-weight, font-size, border-color, width

**Comment by ogchy** [comment]
> These are differences from the first test image
> 
> And these are from the second test image
> 
> You need to pay attention to the: font, font-weight, font-size, border-color, width

image

**Comment by ogchy** [comment]
> These are differences from the first test image
> 
> And these are from the second test image
> 
> You need to pay attention to the: font, font-weight, font-size, border-color, width

ill do it but without button

**Comment by ogchy** [comment]
> These are differences from the first test image
> 
> And these are from the second test image
> 
> You need to pay attention to the: font, font-weight, font-size, border-color, width

can you check

> These are differences from the first test image
> 
> And these are from the second test image
> 
> You need to pay attention to the: font, font-weight, font-size, border-color, width

https://ogchy.github.io/layout_product-cards/ can u check this? something wrong with npm test

File: src/index.html [hunk 5]
```diff
@@ -8,6 +8,50 @@
...
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div>
+        <img
+          src=".//images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        >
+      </div>
```
Comment (volodymyr-soltys97) [inline src/index.html:18]: This div is redundant
```suggestion
      <img
        src=".//images/imac.jpeg"
        alt="imac"
        class="card__image"
      >
```

File: src/index.html [hunk 6]
```diff
@@ -8,6 +8,50 @@
...
+        <h5 class="card__value">$2,199</h5>
+      </div>
+
+      <div>
+      </div>
+
+      <a href="#" data-qa="hover">
+        <button class="card__button">
+          BUY
+        </button>
```
Comment (volodymyr-soltys97) [inline src/index.html:52]: button is redundant here
```suggestion
        BUY
```

File: src/index.html [hunk 7]
```diff
@@ -8,6 +8,50 @@
...
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:35]:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

File: src/index.html [hunk 8]
```diff
@@ -8,6 +8,50 @@
...
+          class="card__image"
+        >
+      </div>
+
+      <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+      <p class="card__product">Product code: 195434</p>
+
+
+      <div class="card__reviews">
```
Comment (volodymyr-soltys97) [inline src/index.html:25]:
```suggestion
      <p class="card__product">Product code: 195434</p>

      <div class="card__reviews">
```

File: src/styles/card.scss [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+    margin: 17px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```
Comment (volodymyr-soltys97) [inline src/styles/card.scss:38]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/styles/card.scss [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+    margin: 0 17px;
+
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+  }
+
+  &__button:hover {
+    color: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/styles/card.scss:78]: Create a variable for repeated color

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Still not fixed from the previous review, you need to fix the failed test
image

Also this part of the task is incomplete
image

**Pull request closed without merging**